### DISCUSSION TITLE
Tiny camera from Seedstudio

### DISCUSSION THREAD

**Discussion by FBMinis** [jameszah/ESP32-CAM-Video-Telegram#44] [General]
https://www.youtube.com/watch?v=8h0iweM5Ngo

https://wiki.seeedstudio.com/xiao_esp32s3_getting_started/

https://files.seeedstudio.com/wiki/SeeedStudio-XIAO-ESP32S3/img/61.gif

**Comment by jameszah** [comment]
Hmmm ... backordered for a month ☹️

**Comment by FBMinis** [comment]
Cool project I just came across that uses it: https://twitter.com/moononournation/status/1651223350914568205?t=CoX6ZFYpnQ7h5xDw5-_xOQ&s=19

**Reply by jameszah** [reply]
Nice - Seed threatened to send me one for "evaluation", but were not impressed with my lack of youtube channel.

Have you seen any problems with telegram recently?  I was sending a jpeg every minute for a few weeks when it stopped working ... text could go through but not binaries??

Going back to the current Brian Lough telegram libraries that now have the certificate.  
Maybe they are throttling me for over use?  

I was unimpressed with my "see in the dark" program so haven't posted it.  Trying a simpler solution.

**Reply by FBMinis** [reply]
I've had 3 cameras working for 7 months now, each one has taken over 6000 photos with no issues of that kind. With my old "send a photoon wake-up" projects, I would receive the text but not the image when sending high-res photos under poor wifi connection or noisy power supply. If things were a bit out of spec, it'd fail. Trying to solve it, I changed from Brian Lough's library to the following one, an those problems became much less frequent: https://github.com/cotestatnt/AsyncTelegram2

I remember feeling the inclusion of certificate made everything happen more straightforward and quicker. But I don't really understand the technical aspect, I just learned this from a lot of trial and error. I think it was also important to leave a small delay between sending the text and the image.

My local electronics shop now carries the Seedstudio camera. But at over 17€ I'll give it a pass for now. https://mauser.pt/catalog/product_info.php?cPath=1667_2889_2891_2888&products_id=095-1672

Regarding the low light conditions, I ended up purchasing a solar rechargeable LED light with PIR from aliexpress for 7€:

https://vi.aliexpress.com/item/1005004971189264.html (got this one because it allows the specific placement of the solar panel)
https://vi.aliexpress.com/item/1005004791450537.html (similar)


![IMG_20230430_170911](https://github.com/jameszah/ESP32-CAM-Video-Telegram/assets/36670323/12231eac-8ecc-4712-ab52-3ce5f7294dbb)
![IMG_20230430_215424](https://github.com/jameszah/ESP32-CAM-Video-Telegram/assets/36670323/78ceb1b0-546f-4ff2-955a-a4a63289ec54)
![IMG_20230430_215439](https://github.com/jameszah/ESP32-CAM-Video-Telegram/assets/36670323/3c0ca29c-5844-4234-869a-3e9859bd72ae)
